VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 | ------- | ----------- | --------------- | -------------------- |
+| 1.3.0-SNAPSHOT   | 3.4.0       | 3.2.0           | 1.2.0                |
```

Comment:
we don't usually modify this file, was it a find-replace thing or do you wanna start doing these so that we don't forget to add the version later on?

Reply:
I thought this would make it easier to keep track of the current iOS, Android and common version of the develop branch.

Reply:
sounds good 👍